packages/agreement-process/src/services/agreementService.ts

```diff
@@ -197,6 +201,25 @@ export function agreementServiceBuilder(
 
       return streamId;
     },
+    async rejectAgreement(
+      agreementId: string,
+      rejectionReason: string,
+      authData: AuthData
+    ): Promise<string> {
+      logger.info("Rejecting agreement");
```

Comment:
For consistency with the Scala codebase, I would also log the agreement id here.

In Scala it does the following:
```scala
val operationLabel = s"Suspending agreement $agreementId"
logger.info(operationLabel)
```

Same for the other logs in this same file